rest/taskana-rest-spring/src/main/java/pro/taskana/rest/ClassificationDefinitionController.java

```diff
@@ -70,14 +71,29 @@
             .collect(Collectors.toMap(i -> i.getKey() + "|" + i.getDomain(), ClassificationSummary::getId));
 
         try {
+            Map<String, String> idMapping = new HashMap<String, String>();
```

Comment:
Das ist prima. An solchen Stellen fände ich es aber noch besser, wenn wir das in sprechende Methoden aufteilen würden, um es verständlicher zu machen. Zum Beispiel:

createOrUpdateClassificationsWithoutParentRelationship(...)
applyParentChildRelationships(...)